regolith/templates/grantreport.txt

```diff
@@ -28,17 +27,39 @@ Goal 4: Apply this to real scientific problems
 
 * What was accomplished under these goals (you must provide information for
 at least one of the 4 categories below)?
-Major Activities:
-Specific Objectives:
-Significant Results:
+------------------------------------------------------------------------------
+- Major Activities (currently worked on projecta):
+PDF in the cloud (PDFitc): Development of a web-based service for determining
+structures from measured PDFs given known structures in structural databases
+xpdTools, xpdAcq, xpdAn, xpdView: Software tools for streaming data analysis
+and visualization of diffraction data from the XPD diffractometer at NSLS-II
+synchrotron. The tools can also be used to analyze data from other diffraction
+beamlines.
+{%- for prum in majorActivities %}
+    {{ prum['name'] }}: {{ prum['description'] }}
+{% endfor -%}
+
+- Specific Objectives:
+PDF in the cloud (PDFitc): Development of a web-based service for determining
+structures from measured PDFs given known structures in structural databases
+
+- Significant Results (finished projecta):
+{%- for prum in significantResults %}
+    {{ prum['name'] }}: {{ prum['description'] }}
+{% endfor -%}
 
 Key outcomes or Other achievements:
 * What opportunities for training and professional development has the project provided?
-{%- for opportunity in presentations %}
-    - {{opportunity -}}
+{%- for opportunity in trainingAndProfessionalDevelopment %}
+    - Authors: {{ opportunity['authors'] }}
+    - Title: {{ opportunity['title'] }}
+    - Location: {{ opportunity['meeting_name'] }}
 {% endfor -%}}
 
 * How have the results been disseminated to communities of interest?
+1. Publications
```

Comment:
This is an interesting one.   It should probably be in people too so any user of Regolith can decide for themselves.